testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/ConfigurationTools.cs

```diff
@@ -0,0 +1,90 @@
+using UnityEngine;
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Unity.Netcode.MultiprocessRuntimeTests
+{
+    public class ConfigurationTools
+    {
+        public static JobQueueItemArray GetJobQueue()
+        {
+            var theList = new JobQueueItemArray();
+
+            var t = GetRemoteConfig();
+            t.Wait();
```

Comment:
will you still get bubbled up exceptions with this?

Reply:
No, I struggle with how to do this properly but I've made one more edit that I think gets to what you are looking for, if this doesn't solve it then please point me to an example that might follow to get this done correctly.

Reply:
Alright, so you're effectively blocking Unity's main thread while waiting for a response from your HTTP server. This in itself is ok thinking of it, since you don't want your tests to start until you have all your configuration. Some things you might want to take into account:
- Calling wait with a timeout in its parameter.
- Trying to wait as little as possible and use async await instead (and wait only when necessary). So you'd have async method1() { await method2(); }; async method2() {await method3(); } etc.
- from this https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.wait?view=net-6.0 Wait will throw an exception if something wrong happens during the execution of the method. However, that exception is wrapped. So what you'll see in your logs is a stacktrace starting at t.Wait instead of the actual internal exception. You'll want to log that internal exception instead. Using async await like in my second point above will bubble up that inner exception, which means you don't have anything to do with it. You only need to handle it when you Wait basically (hence why using as little Wait as possible)